login.php

```diff
@@ -62,8 +62,28 @@
 
 		<!-- This will have an image slider -->
 		<div class="slider">
-			<h1>Login to see all of Graeme's tunes</h1>
-			<p>Placeholder for an image slider</p>
+			<h1 class="slider_heading">Login to see all of Graeme's tunes</h1>
```

Comment:
The image slideshow has been added here, this heading now needs to be displayed over the images